awe_languagetool/languagetoolServer.py

```diff
@@ -9,14 +9,15 @@
 this repository for details.
 '''
 
+import argparse
 import os
 import subprocess
 import awe_languagetool
 
 from importlib import resources
 
 
-def runServer(fileName=None, port=8081, config_file="languagetool.cfg"):
+def runServer(port=8081, config_file=None):
```

Comment:
Not a showstopper on merge, but I feel this would do better broken out into functions. In pseudocode:

```
function run_server()
   config_file = find_config_file(..)
   config_file_param = f"--config {config_file}" if config_file else None
   ltc = "java -cp {LANGUAGE_TOOL_PATH} \
             {config_file_param}
            org.languagetool.server.HTTPServer \
            --port {port} --allow-origin \"*\""
  runner = subprocess.Popen
  if not runner
     raise error
```

And perhaps similar for parsing arguments in `__name__==__main__`. This code is a little bit long, and has multiple functions confounded together.